framework/wazuh/core/cluster/master.py

```diff
@@ -301,7 +301,7 @@ def process_request(self, command: bytes, data: bytes) -> Tuple[bytes, bytes]:
         else:
             return super().process_request(command, data)
 
-    async def execute(self, command: bytes, data: bytes, wait_for_complete: bool) -> Dict:
+    async def execute(self, command: bytes, data: bytes, wait_for_complete: bool = True) -> Dict:
```

Comment:
The default value should be False since it must behave as if it does not exist. 
```suggestion
    async def execute(self, command: bytes, data: bytes, wait_for_complete: bool = False) -> Dict:
```